### ISSUE TITLE
Print work directory when a task execution is retried

### ISSUE THREAD

**Issue by skptic** [nextflow-io/nextflow#445]
Currently when a task fails, the work directory is not provided.

Current output:
```
WARN: Process `dpa_alignment (Ald_Xan_dh_2 - UPP - DPA - 250)` terminated with an error exit status (126) -- Execution is retried (1)
```

With  many tasks being executed, it makes it very hard to to trace back where the task work directory is.

As a solution, the work directory could be printed. To be consistent with submission:
```
[wo/rkdire] WARN: Process `dpa_alignment (Ald_Xan_dh_2 - UPP - DPA - 250)` terminated with an error exit status (126) -- Execution is retried (1)
```

**Comment by pditommaso** [comment]
Included in version `0.25.8-SNAPSHOT`

**Comment by pditommaso** [comment]
I've changed this message from `WARN` to a `NOTE` as shown below: 

```
[26/5dc6d8] NOTE: Error submitting process 'foo' for execution -- Execution is retried (1)
[7d/c742e2] NOTE: Error submitting process 'foo' for execution -- Execution is retried (2)
```

It should be fine.

**Comment by pditommaso** [comment]
Included in version `0.26.0-beta5`
